### DISCUSSION TITLE
No BlobContainerName found in Keda Operator Logs

### DISCUSSION THREAD

**Discussion by kmatthews3-zz** [kedacore/keda#942] [General]
We have a function app which works locally in the dockerfile that is generated, however when we run the func kubernetes deploy command, the scaled object does not have the required blobContainerName metadata and therefore does not scale.

Where do we pass the BlobContainerName Metadata in a DotNet Application?

Thanks
Kevin

**Comment by tomkerkhove** [comment]
This looks like an Azure Functions issue, can you report it on https://github.com/Azure/azure-functions-core-tools please?

They will have to provide it during `func kubernetes deploy`

/cc @ahmelsayed
